lib/helper/WebDriverIO.js

```diff
@@ -1877,16 +1881,11 @@ async function forEachAsync(array, callback, option = {}) {
   const inputArray = Array.isArray(array) ? array : [array];
   const values = [];
   for (let index = 0; index < inputArray.length; index++) {
-    let res;
-    try {
-      res = await callback(inputArray[index], index, inputArray);
-      if (Array.isArray(res) && expandArrayResults) {
-        res.forEach(val => values.push(val));
-      } else if (res) {
-        values.push(res);
-      }
-    } catch (err) {
-      throw err;
```

Comment:
don't we need the error catching anymore?

Reply:
oh nvm, we will drop this helper at some point.

Reply:
we catch `catch (err)` and then throw again `throw err` without any change. So this catch is useless